### PR TITLE
Allow full screen by default

### PR DESCRIPTION
**Pull request by adityaruplaha**
Will add a description soon.

---
This change is [Reviewable](https://reviewable.io/reviews/citra-emu/citra/3570)

### CODE REVIEW

File: src/citra/config.cpp [hunk 1]
```diff
@@ -93,6 +93,10 @@ void Config::ReadValues() {
     // Core
     Settings::values.use_cpu_jit = sdl2_config->GetBoolean("Core", "use_cpu_jit", true);
 
+    // Window Options
+    Settings::values.use_cpu_jit =
```
Comment (chris062689) [inline src/citra/config.cpp]: Why are we using use_cpu_jit as a variable container for defaulting to full screen? I assume this is not intended.

**Comment by chris062689** [comment]
This should be a parameter when starting both citra.exe and citra-qt.exe
I don't see the use in creating a variable for it. This would only confuse users.

File: src/citra_qt/main.cpp [hunk 2]
```diff
@@ -934,12 +936,12 @@ void GMainWindow::ToggleFullscreen() {
 
 void GMainWindow::ShowFullscreen() {
     if (ui.action_Single_Window_Mode->isChecked()) {
-        UISettings::values.geometry = saveGeometry();
+        // UISettings::values.geometry = saveGeometry();
```
Comment (CodingKoopa) [inline src/citra_qt/main.cpp]: Instead of commenting these lines out, they should just be removed.

**Comment by jroweboy** [comment]
Please take more effort to test your PR before opening it. No description, copy paste errors, commenting lines out instead of removing them, and last of all, you removed the lines that sets/restores window geometry, meaning citra no longer remembers window position or size between launches. When you fix all of these things, feel free to open this PR, but right now its not ready for review.

File: src/citra_qt/main.cpp (hunk 2 above)
Comment (adityaruplaha) [reply src/citra_qt/main.cpp]: Will do. Was checking if it caused any errors, & then forgot to remove.

File: src/citra/config.cpp (hunk 1 above)
Comment (adityaruplaha) [reply src/citra/config.cpp]: This just proves how careless I can be. Extremely sorry, will fix.

**Comment by adityaruplaha** [comment]
I should have added a WIP label. And I know my PR is far from complete. I am still thinking abt that geometry thing, but everything else should have been ready. I apologise.

**Comment by adityaruplaha** [comment]
And, I don't think a variable would be confusing here. (I actually asked a few users about this option, and they didn't have any trouble understanding what I meant). About that SDL2 option, might take me some time (maybe in a seperate PR?) as I'm unfamiliar with that part. And thanks for the quick review.

File: src/citra_qt/main.cpp (hunk 2 above)
Comment (adityaruplaha) [reply src/citra_qt/main.cpp]: This code has been rewritten.

File: src/citra/config.cpp (hunk 1 above)
Comment (adityaruplaha) [reply src/citra/config.cpp]: Fixed.

**Comment by adityaruplaha** [comment]
Ok, I think I've ironed out all the bugs. Will do a bit of testing and open a new PR.

**Pull request closed without merging**